lib/src/models/tx_output.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:ledger_cardano/src/models/asset_group.dart';
+import 'package:ledger_cardano/src/models/datum.dart';
+import 'package:ledger_cardano/src/models/tx_output_destination.dart';
+import 'package:ledger_cardano/src/utils/constants.dart';
+
+part 'tx_output.freezed.dart';
+
+@freezed
+sealed class TxOutput with _$TxOutput {
+   TxOutput._();
+
+  factory TxOutput.alonzo({
+    required TxOutputDestination destination,
+    required BigInt amount, // Using String to represent bigint_like
+    List<AssetGroup>? tokenBundle,
+    @Default(null) String? datumHashHex,
```

Comment:
Don't add `@Default(null)`. It's already null by default. The `@Default` annotation is required only when you have a non-nullable type without the `required` keyword.